Services/HandlerManager.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+
+namespace Symfony\Bundle\MonologBundle\Services;
```

Comment:
We usually don't use `Services` namespace. Can you put this class in `Symfony\Bundle\MonologBundle` instead?